File: go-sdk/pkg/models/operations/getanoauthapplication.go
package operations

type GetAnOauthApplicationPathParams struct {
	ApplicationID string `pathParam:"style=simple,explode=false,name=application_id"`
	Organization  string `pathParam:"style=simple,explode=false,name=organization"`
}

type GetAnOauthApplicationRequest struct {
	PathParams GetAnOauthApplicationPathParams
}

type GetAnOauthApplicationResponse struct {
	ContentType                                   string
	StatusCode                                    int
	GetAnOauthApplication200ApplicationJSONObject map[string]map[string]interface{}
}

File: ts-sdk/src/sdk/models/operations/getanoauthapplication.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { Type } from "class-transformer";


export class GetAnOauthApplicationPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=application_id" })
  applicationId: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;
}

export class GetAnOauthApplicationRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetAnOauthApplicationPathParams;
}

export class GetAnOauthApplicationResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  getAnOauthApplication200ApplicationJSONObject?: Record<string, Record<string, any>>;
}